valhalla/jawn/src/managers/score/ScoreManager.ts

```diff
@@ -0,0 +1,59 @@
+import { dbExecute } from "../../lib/shared/db/dbExecute";
+import { err, Result } from "../../lib/shared/result";
+import { BaseManager } from "../BaseManager";
+
+export class ScoreManager extends BaseManager {
+  public async addScores(
+    requestId: string,
+    scores: Record<string, number>
+  ): Promise<Result<string, string>> {
+    try {
+      const organizationId = this.authParams.organizationId;
+      const scoreKeys = Object.keys(scores);
+      const scoreValues = Object.values(scores);
+
+      const { error: requestError } = await dbExecute(
+        `SELECT id FROM request WHERE id = $1 AND helicone_org_id = $2`,
+        [requestId, organizationId]
+      );
+
+      if (requestError) {
+        return err(`${requestId} not found in organization ${organizationId}`);
+      }
+
+      const organizationIds = Array(scoreKeys.length).fill(organizationId);
+
+      const query = `
+        WITH upserted_attributes AS (
+            INSERT INTO score_attribute (score_key, organization)
+            SELECT unnest($1::text[]), unnest($2::uuid[])
+            ON CONFLICT (score_key, organization) DO UPDATE SET
+                score_key = EXCLUDED.score_key
+            RETURNING id
+        ),
+        inserted_values AS (
+            INSERT INTO score_value (score_attribute, request_id, int_value)
+            SELECT id, $3, unnest($4::bigint[])
+            FROM upserted_attributes
+            RETURNING *
+        )
+        SELECT * FROM inserted_values;
```

Comment:
Why are you returning things?

Reply:
@chitalian , @colegottdank I addede select because there there is not transaction commits before. Without this select I got `error: syntax error at end of input`. 

we can do like:

` await client.query('BEGIN');
  query execution ...
  await client.query('COMMIT');
`